src/games/strategy/triplea/ResourceLoader.java

```diff
@@ -98,14 +99,16 @@ public boolean match(final File f) {
         return f.exists();
       }
     });
-    if (existing.size() > 1) {
-      System.out.println("INFO: Found too many files for: " + mapName + "  found: " + existing);
-    }
+
+
     // At least one must exist
     if (existing.isEmpty()) {
       return Collections.emptyList();
```

Comment:
`Collections.emptyList();` is not guava. When the intent is for an empty collection, getting one from `Collections` is preferred to new'ing one up. It's similar to preferring `collection.isEmpty()` over `collection.size() > 0`